### ISSUE TITLE
Feature Request: Option to [permanently] move apps from the short-list to the "More Apps" or some other folder

### ISSUE THREAD

**Issue by dasnorsep** [kyleneideck/BackgroundMusic#292]
Hey there!

Thank you so much the app!

The only issue I have is 18 apps running at all times and it's a tiny bit annoying to see all of them in the drop-down menu. I would really appreciate an option to hide apps in the "More Apps" or maybe another, user-created folder. It would be so much more aesthetically pleasing for me and other users, I'm sure :3 

Thanks ahead!

**Comment by maxwell-01** [comment]
+1 on this. I really don't need activity monitor and excel competing for the top places with my browser and music players!

**Comment by Brog33** [comment]
yeah the list is flooded with app processes that don't use audio or is irrelevan. since i can't get rid of them the app is basically useless as a quick volume control.
